flake.nix

```diff
@@ -0,0 +1,115 @@
+{
+  description = "Nix package for centrifuge-chain";
+
+  inputs = {
+    nixpkgs.url = github:NixOS/nixpkgs/nixos-20.09;
+    gitignore = {
+      url = github:hercules-ci/gitignore.nix;
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+
+  };
+
+  outputs = inputs:
+    let
+      name = "centrifuge-chain";
+      major = "2.0.0";
```

Comment:
Please match the node version https://github.com/centrifuge/centrifuge-chain/blob/parachain/Cargo.toml#L3